core/src/main/java/de/danoeh/antennapod/core/preferences/UserPreferences.java

```diff
@@ -264,6 +269,26 @@ public static boolean getUseEpisodeCoverSetting() {
         return prefs.getBoolean(PREF_USE_EPISODE_COVER, true);
     }
 
+    /**
+     * @return {@code true} if we should show remaining time or the duration
+     */
+    public static boolean getShowRemainTimeSetting() {
+        return prefs.getBoolean(PREF_SHOW_TIME_LEFT, false);
+    }
+
+    /**
+     * Sets the preference for whether we show the remain time, if not show the duration. This will
+     * send out events so the current playing screen, queue and the episode list would refresh
+     *
+     * @return {@code true} if we should show remaining time or the duration
+     */
+    public static void setShowRemainTimeSetting(Boolean showRemain) {
+        prefs.edit().putBoolean(PREF_SHOW_TIME_LEFT, showRemain).apply();
+        EventBus.getDefault().post(new ShowRemainTimeUpdateEvent());
+        EventBus.getDefault().post(new UnreadItemsUpdateEvent());
+        EventBus.getDefault().post(new PlayerStatusEvent());
```

Comment:
UserPreferences currently does not send events. Could you please send it in the preferences fragment instead?

Reply:
done